src/getMachines.py

```diff
@@ -0,0 +1,51 @@
+import boto3
+from decEncoder import *
+
+"""Scan dynamo table"""
+dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')
+machineTable = dynamodb.Table('Machines')
+response = machineTable.scan()
```

Comment:
Mentioned this in Slack, but this won't be updated until probably 10-15 minutes after the function is last called. And if it's continuously called within about 10 minutes then it will never update. Do this in the handler function and add filters so the work is done more on the Dynamo side.